spec/features/user_asks_question_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+describe "ask a question", type: :feature do
```

Comment:
Another way to do this if you want to use capybara is to use the `feature` `scenario` syntax. It's just like `describe` `it` but capybara is aware of it.

something like
```ruby
feature "ask a question" do
  scenario "user sees posted question" do
     #....
  end
end
```